sale_partner_shipping_invoice_domain/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Adds a domain on shipping address to filter out the irrelevant contacts instead of choosing from every contact in DB.
```

Comment:
Adds a domain on shipping and invoicing address in SO to only allow partner's child contacts instead of choosing from every contact in DB.